src/lib/helpers/write-csv-to-s3.js

```diff
@@ -41,12 +41,12 @@ module.exports = {
 
       return fs.unlink(filePath, err => {
         if (err) {
-          logger.error(new Error(err))
+          logger.error(new Error(err.stack))
```

Comment:
Not a thing you've done @Beckyrose200 . But curious as to why they took an error and then wrapped it in a `new Error()`.

I suspect we could simplify things

```suggestion
          logger.error(err.stack)
```

Reply:
Made the above changes and the tests still pass :) (although there is only 30 of them)